content/docs/command-reference/status.md

```diff
@@ -34,17 +35,18 @@ options:
 | remote | `--cloud`      | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |
 
 DVC determines which data and code files to compare by analyzing all
-[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr> (the
-`--all-branches` and `--all-tags` options compare multiple workspace versions).
+[`.dvc` files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr>
```

Comment:
analyzing all stages (in `dvc.yaml`) and `.dvc` files

Reply:
This change applies all over this doc. Please try to generalize this in this file. Status checks stages (in dvc.yaml files) as well as .dvc files. Thanks

Reply:
I checked status quickly and I don't see any more mentions of dvc.yaml. Please try to use "dvc.yaml" AND/OR ".dvc file(s)" as it seems appropriate in every comparison.

Reply:
And request my review when ready. Thanks

Reply:
Still no changes to status... I'll make a specific review.